bundles/io/org.openhab.io.transport.cul.test/src/test/java/org/openhab/io/transport/cul/CulConstructorsTest.java

```diff
@@ -0,0 +1,57 @@
+package org.openhab.io.transport.cul;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import junit.framework.Assert;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.openhab.io.transport.cul.internal.CULNetworkHandlerImpl;
+import org.openhab.io.transport.cul.internal.CULSerialHandlerImpl;
+
+public class CulConstructorsTest {
+	
+// code below needed if testing with proper mocks for devices
+//	@Before
+//	public void initHandlers(){
+//		CULManager.registerHandlerClass("serial", CULSerialHandlerImpl.class);
+//		CULManager.registerHandlerClass("network", CULNetworkHandlerImpl.class);
+//	}
+	
+	@Test
+	public void testCreateSerialHandler() throws Exception{
+		CULHandler handler=new CULSerialHandlerImpl("myDevice",CULMode.SLOW_RF);
+
+		Assert.assertNotNull(handler);
+		//will not work w/o mocking
+		//handler=CULManager.getOpenCULHandler("serial:/dev/somedummyserialdevice", CULMode.SLOW_RF);

```

Comment:
in order to test arePropertiesEqual() it doesn't seem to be necessary to work with a CulManager. This code can be removed though (and the initHandles() as well)